Rubberduck.Main/VbeProvider.cs

```diff
@@ -24,7 +24,6 @@ internal static void Initialize(IVBE vbe)
 
         internal static void Terminate()
         {
-            Vbe.Dispose();
```

Comment:
Might be useful to add a comment explaining that the Provider doesn't actually own these references to avoid someone reintroducing this

Reply:
I'm sure that one got flagged by the analyzer I was experimenting with. Since the analyzer does not allow annotating (yet), I probably made the mistake of thinking it had to be disposed because other methods did need to dispose since they got a new SCW. This did not, however. Apologies.